.github/workflows/realm-dart-linux.yml

```diff
@@ -2,19 +2,22 @@ name: Realm Dart for Linux
 
 on:
   push:
-    branches: 
+    branches:
       - master
-    # Do not run if change is in these paths
     paths-ignore:
       - 'flutter/**'
       - 'generator/**'
+      - '.github/**'
```

Comment:
Why not run on changes in these files? Isn't it required to run CI when changing CI scripts?

Reply:
so this ignores everything in github except linux workflow?

Reply:
Yes